apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -402,12 +402,12 @@ schema("/nodes/:node/bridges/:id/operation/:operation") ->
 '/bridges'(get, _Params) ->
     {200,
         zip_bridges([
-            [format_resp(Data, Node) || Data <- emqx_bridge_proto_v1:list_bridges(Node)]
+            [redact(format_resp(Data, Node)) || Data <- emqx_bridge_proto_v1:list_bridges(Node)]
          || Node <- mria_mnesia:running_nodes()
         ])}.
 
 '/bridges/:id'(get, #{bindings := #{id := Id}}) ->
-    ?TRY_PARSE_ID(Id, lookup_from_all_nodes(BridgeType, BridgeName, 200));
+    ?TRY_PARSE_ID(Id, redact(lookup_from_all_nodes(BridgeType, BridgeName, 200)));
```

Comment:
I think that those calls to `redact` would be better inserted at `format_resp` and `format_bridge_info`, since we'll probably want that effect every time we format data for a response.